Tasks/MSBuildV1/MSBuild.ps1

```diff
@@ -19,7 +19,7 @@ try {
     [bool]$maximumCpuCount = Get-VstsInput -Name MaximumCpuCount -AsBool
     [bool]$restoreNuGetPackages = Get-VstsInput -Name RestoreNuGetPackages -AsBool
     [bool]$logProjectEvents = Get-VstsInput -Name LogProjectEvents -AsBool
-    [bool]$createLogFile = (Get-VstsInput -Name CreateLogFile -AsBool) -or $debug
+    [bool]$createLogFile = (Get-VstsInput -Name CreateLogFile -AsBool)
```

Comment:
Could you please apply the same changes for VSBuild task also?

Reply:
Done and tested:
https://dev.azure.com/v-andreym/MSBuild%20Test%20Project/_build/results?buildId=93&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9
https://dev.azure.com/v-andreym/MSBuild%20Test%20Project/_build/results?buildId=95&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9